sklearn/decomposition/pca.py

```diff
@@ -120,6 +120,10 @@ class PCA(_BasePCA):
     Notice that this class does not support sparse input. See
     :class:`TruncatedSVD` for an alternative with sparse data.
 
+    PCA uses the maximum likelihood estimate of the eigenvalues, which does not
```

Comment:
I think this is fairly minor, so create a Notes section below where this can be included.